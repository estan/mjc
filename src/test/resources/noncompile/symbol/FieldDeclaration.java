class FieldDeclarationTest {
    public static void main(String[] args) {
    }
}
class A {
    int a;
    long a; // DUPLICATE_FIELD
    B b; // UNDECLARED_CLASS
}
