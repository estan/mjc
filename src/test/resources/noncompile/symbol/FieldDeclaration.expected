DUPLICATE_FIELD
UNDECLARED_CLASS
